telegram/_utils/enum.py

```diff
@@ -23,9 +23,13 @@
     user. Changes to this module are not considered breaking changes and may not be documented in
     the changelog.
 """
-from enum import Enum
+import sys
+from enum import Enum, IntEnum
```

Comment:
`import enum` and then use ase `enum.(IntEnum)` -> fixes the "redef" thingy below

Reply:
ah wow